Serverp/src/ServerThread.java

```diff
@@ -0,0 +1,5 @@
+
+public class ServerThread
```

Comment:
yeah I think it should be:

public class ServerThread extends Thread

or 

public class ServerThread implements Runnable

Depending on what you need, I prefer the second personally as you can inherit only one class but implement many interfaces.